src/redux/books/books.js

```diff
@@ -1,7 +1,18 @@
 const BOOK_ADDED = 'bookstore/books/BOOK_ADDED';
 const BOOK_REMOVED = 'bookstore/books/BOOK_REMOVED';
 
-const initialState = [];
+const initialState = [
+  {
+    id: '0',
+    title: 'Book 1',
+    author: 'Author 1',
+  },
+  {
+    id: '1',
+    title: 'Book 2',
+    author: 'Author 2',
+  },
```

Comment:
Great job @gandradep ✋

- [ ] You may kindly remove static data from your project since you will be generating data from the form.
- [ ] As stated in the project requirements, kindly make sure the `Remove Book` button works as expected.

Reply:
@1GORDON thank you for the review. 
The static data is a project requirement. 

> Secondly, change the default state in your books reducer from an empty array to an array with a few books. One of the goals is to display them in your React components.
About the Remove book I already fixed it, I was looking for a number id and it has a string.